bases/world_spi/global_indicators/table_config.yaml

```diff
@@ -0,0 +1,1351 @@
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: world_spi
+
+table_id: global_indicators
+
+# Título da tabela.
+title: Social Progress Index
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: The Social Progress Index insights visualization gives you a holistic understanding of world progress from 2011-2022, as well as a granular set of information.
```

Comment:
Você pode descrever mais o que a pessoa irá encontrar na tabela em especifico, como ex "The table uses 12 components and 60 indicators to measure the social performance of 169 countries fully and an additional 27 countries partially."